setup.py

```diff
@@ -29,19 +29,24 @@ def run(self):
         raise SystemExit(errno)
 
 
-setup(name='sklearn-pandas',
+setup(name='sklearn-pandas2',
       version=__version__,
       description='Pandas integration with sklearn',
-      maintainer='Israel Saeta Pérez',
-      maintainer_email='israel.saeta@dukebody.com',
-      url='https://github.com/paulgb/sklearn-pandas',
+      maintainer='Ritesh Agrawal',
+      maintainer_email='ragrawal@gmail.com',
+      url='https://github.com/scikit-learn-contrib/sklearn-pandas',
       packages=['sklearn_pandas'],
       keywords=['scikit', 'sklearn', 'pandas'],
       install_requires=[
-          'scikit-learn>=0.15.0',
-          'scipy>=0.14',
-          'pandas>=0.11.0',
-          'numpy>=1.6.1'],
+          'scikit-learn>=0.23.0',
+          'scipy>=1.4.1',
+          'pandas>=1.0.5',
+          'numpy>=1.18.1',
+          'tqdm>=4.46.0'
```

Comment:
If we put `tqdm` in `extras_require` we have to delete it from `install_requires` here.